manager/integration/tests/test_ha.py

```diff
@@ -859,4 +860,75 @@ def test_single_replica_failed_during_engine_start():
     11. Check if the volume still works fine by
         r/w data and creating/removing snapshots.
     """
-    pass
+    pv_name = "pv-" + volume_name
+    pvc_name = "pvc-" + volume_name
+    pod_name = "pod-" + volume_name
+
+    volume = create_and_check_volume(client, volume_name)
+    create_pv_for_volume(client, core_api, volume, pv_name)
+    create_pvc_for_volume(client, core_api, volume, pvc_name)
+
+    pod['metadata']['name'] = pod_name
+    pod['spec']['volumes'] = [{
+        'name': pod['spec']['containers'][0]['volumeMounts'][0]['name'],
+        'persistentVolumeClaim': {
+            'claimName': pvc_name,
+        },
+    }]
+
+    create_and_wait_pod(core_api, pod)
+    wait_for_volume_healthy(client, volume_name)
```

Comment:
You can use `prepare_pod_with_data_in_mb()` in the beginning. This modification/suggestion is optional for you.